client_admin/pulp/client/admin/tasks.py

```diff
@@ -231,12 +247,27 @@ def retrieve_tasks(self):
 class AllTasksSection(BaseTasksSection):
     FIELDS = ('tags', 'task_id', 'state', 'start_time', 'finish_time')
 
+    def __init__(self, context, name, description):
+        BaseTasksSection.__init__(self, context, name, description)
+
+        self.list_command.add_option(self.all_flag)
```

Comment:
It would be better to add these options right after you create the list command, so everything is organized in one place. I don't see value in waiting to do it here.